src/FondOfSpryker/Yves/EnhancedEcommerce/Theme/default/partials/data-layer.twig

```diff
@@ -0,0 +1,5 @@
+<script>
+    window.dataLayer = window.dataLayer || [];
+
+    {{ dataLayerString|raw }}
```

Comment:
Is this correct printing a plain string in script ? An object seems more logical, what's behind that dataLayerString variable?

Reply:
jepp, enhanced ecommerce is quite more then rendering a simple datalayer object. in some cases we need different renderings for the connector-modules (i.e. for javascript-stuff). before it was returning an array which was only rendering the datalayer. now the connector modules are returning complete rendered templates as string. the name "dataLayerString" is a little bit confusing, maybe javascript/javascriptString or something is a better choice.

Reply:
Okay